src/components/Selectors/CheckBox/CheckBox.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC } from 'react';
+import { CheckBoxProps, SelectorSize } from '../Selectors.types';
+import { classNames } from '../../../shared/utilities';
+
+import styles from '../selectors.module.scss';
+
+export const CheckBox: FC<CheckBoxProps> = ({
+    ariaLabel,
+    checked = false,
+    color,
+    defaultChecked = false,
+    disabled = false,
+    inputType = 'checkbox',
+    id,
+    name,
+    size = SelectorSize.Medium,
+    value,
+    onClick,
+}) => {
+    const checkBoxDivClassNames: string = classNames([styles.selector]);
+
+    const checkBoxInputWrapperClassNames: string = classNames([
+        styles.selectorWrapper,
+    ]);
+
+    const checkBoxLabelClassNames: string = classNames([styles.selectorLabel]);
```

Comment:
this is redundant if we are not merging classes just add the style directly on the item, same for line 20, 22